src/ServiceInsight/Models/StoredMessage.cs

```diff
@@ -19,9 +19,7 @@ public StoredMessage()
         public MessageIntent MessageIntent { get; set; }
         public Endpoint SendingEndpoint { get; set; }
         public Endpoint ReceivingEndpoint { get; set; }
-        public TimeSpan CriticalTime { get; set; }
         public TimeSpan ProcessingTime { get; set; }
-        public TimeSpan DeliveryTime { get; set; }
         public string ConversationId { get; set; }
 
         public MessageStatistics Statistics
```

Comment:
If it is not in use, u can delete it

On Thursday, July 23, 2015, Weronika Łabaj <notifications@github.com> wrote:

> In src/ServiceInsight/Models/StoredMessage.cs
> <https://github.com/Particular/ServiceInsight/pull/477#discussion_r35312403>
> :
>
> >          public TimeSpan ProcessingTime { get; set; }
> > -        public TimeSpan DeliveryTime { get; set; }
> >          public string ConversationId { get; set; }
> >
> >          public MessageStatistics Statistics
>
> @distantcam <https://github.com/distantcam> I think that this class is
> not used anywhere at the moment, can we get rid of it or did I miss
> something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/ServiceInsight/pull/477/files#r35312403>.
>